src/paging.rs

```diff
@@ -545,12 +549,16 @@ impl<T: Translation> PageTableWithLevel<T> {
                     i += 1;
                 }
                 if i - 1 == first_zero {
-                    writeln!(f, "{:indentation$}{}: 0", "", first_zero)?;
+                    writeln!(f, "{:indentation$}{: <WIDTH$}: 0", "", first_zero)?;
```

Comment:
This change looks unrelated, maybe it should be a separate commit.

Reply:
Moved to a new commit.